x-pack/plugin/otel-data/src/main/resources/component-templates/traces-otel@mappings.yaml

```diff
@@ -0,0 +1,70 @@
+---
+version: ${xpack.oteldata.template.version}
+_meta:
+  description: Default mappings for OpenTelemetry traces
+  managed: true
+template:
+  mappings:
+    date_detection: false
+    properties:
+      data_stream.namespace:
```

Comment:
```suggestion
      data_stream.type:
```

Reply:
Also why is this specified here, if it is already specified in traces@mappings.yaml?

Reply:
Thanks for catching this. 

> Also why is this specified here, if it is already specified in [traces@mappings.yaml](mailto:traces@mappings.yaml)?

I suspect that is because here we set `value: traces`, which was not (or should have not been) present on `traces@mappings`. For now I just removed this part. Let's discuss if we want to have it here or in `traces@mappings`.